### PR TITLE
Fix wrap() for missing positional argument problem in Geti

### PR DESCRIPTION
**Pull request by vinnamkim**
### Summary
 - This patch passes Geti api_on_merge_request: 
1. http://validationreports.sclab.intel.com:8004/reports/build_number_report?test_session_build_number=IMPTOps-59349&environment=mclx77
2. https://ci.iotg.sclab.intel.com/job/IMPT/job/IMPTOps/59349/

### How to test
I added a unit test for this change

### Checklist

- [ ] I have added unit tests to cover my changes.​
- [ ] I have added integration tests to cover my changes.​
- [ ] I have added e2e tests for validation.
- [ ] I have added the description of my changes into CHANGELOG in my target branch (e.g., [CHANGELOG](https://github.com/openvinotoolkit/training_extensions/blob/develop/CHANGELOG.md) in develop).​
- [ ] I have updated the documentation in my target branch accordingly (e.g., [documentation](https://github.com/openvinotoolkit/training_extensions/tree/develop/docs) in develop).
- [ ] I have [linked related issues](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

### License

- [ ] I submit _my code changes_ under the same [Apache License](https://github.com/openvinotoolkit/training_extensions/blob/develop/LICENSE) that covers the project.
      Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below).

```python
# Copyright (C) 2023 Intel Corporation
# SPDX-License-Identifier: Apache-2.0
```

### CODE REVIEW

File: otx/api/entities/dataset_item.py [hunk 1]
```diff
@@ -493,14 +494,14 @@ def get_metadata_by_name_and_model(self, name: str, model: Optional[ModelEntity]
...
         """Creates a new DatasetItemEntity, overriding only the given arguments to the existing ones for this instance."""
-        return self.__class__(
-            media=kwargs.get("media", self.media),
-            annotation_scene=kwargs.get("annotation_scene", self.annotation_scene),
-            roi=kwargs.get("roi", self.roi),
-            metadata=kwargs.get("metadata", self.get_metadata()),
-            subset=kwargs.get("subset", self.subset),
-            ignored_labels=kwargs.get("ignored_labels", self.ignored_labels),
-        )
+        params = {
```
Comment (eunwoosh) [inline otx/api/entities/dataset_item.py:497]: I think metadata can't be passed to new class becuase metadata doesn't have property or setter. Is it ok?

**Review by goodsong81 — DISMISSED** [review]
Thank you for the quick fix!

File: otx/api/entities/dataset_item.py (hunk 1 above)
Comment (vinnamkim) [reply otx/api/entities/dataset_item.py:497]: Thanks, it should be fixed

Comment (vinnamkim) [reply otx/api/entities/dataset_item.py:497]: Fixed in https://github.com/openvinotoolkit/training_extensions/pull/2004/commits/3b1e636fcbc92b0b0fc99b299e82b45d0019862d.

**Review by eunwoosh — APPROVED** [review]

**Comment by vinnamkim** [comment]
Re-run Geti build: https://ci.iotg.sclab.intel.com/job/IMPT/job/IMPTOps/59463/

**Comment by vinnamkim** [comment]
@goodsong81 

There is no failure: http://validationreports.sclab.intel.com:8004/reports/build_number_report?test_session_build_number=IMPTOps-59463&environment=mclx77

**Review by sungmanc — APPROVED** [review]

**Review by goodsong81 — APPROVED** [review]

**Pull request merged**